katello/katello-backup

```diff
@@ -35,6 +63,22 @@ optparse = OptionParser.new do |opts|
     @options[:logical_backup] = logical
   end
 
+  opts.on("--snapshot", "Locations of the logical volumes of #{DATABASES.join(',')}") do |snapshot|
+    @options[:snapshot] = snapshot
+  end
+
+  opts.on("--snapshot-size SNAPSHOT_BLOCK_DEVICE_SIZE", String, "Override default block size (5G)") do |size|
+    @options[:snapshot_size] = size
+  end
+
+  opts.on("--snapshot-filetype FILE_SYSTEM_TYPE", String, "Override default filesystem type (xfs)") do |type|
```

Comment:
However, if the file type is xfs, there needs to be the -onouuid,ro tag added. Without it, mounting fails.